### ISSUE TITLE
Duplicated licenses 

### ISSUE THREAD

**Issue by ma2ciek** [ckeditor/ckeditor5-build-classic#2]
I see a lot of:

```
 * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
 * For licensing, see LICENSE.md.
```

especially in the [es6 build](https://github.com/ckeditor/ckeditor5-build-classic/blob/0bf5c2d4e9136194451e9faf7e6ef2f23d71fc79/build/ckeditor.es6.js) where the license is added per each file in the project. Can we remove them and add one at the top to optimise the file size?

**Comment by pomek** [comment]
Guys, I'm still working on this code…

![image](https://cloud.githubusercontent.com/assets/2270764/23998354/2377b616-0a55-11e7-846a-a0cbb00ea508.png)

**Comment by ma2ciek** [comment]
Don't take it to personally :D Now you have an option to close one more ticket when you finish the job ;) It's the result of the webpack builder which I was also implementing so it's particularly my oversight ;)

**Comment by pomek** [comment]
To be honest, this ticket should be reported as a comment here https://github.com/ckeditor/ckeditor5-dev/issues/131 :D

**Comment by pomek** [comment]
Options for removing comments from compiled code does not work… @Reinmar noticed it earlier https://github.com/babel/babili/issues/396#issuecomment-276644610.

**Comment by ma2ciek** [comment]
So it could be a part of webpack / rollup compilation then. Maybe not the best option, but in the end it may work.

**Comment by Reinmar** [comment]
Will be fixed by #3.

**Comment by Reinmar** [comment]
Or actually, it was fixed by https://github.com/ckeditor/ckeditor5-dev/commit/a695ac50a3e251d549fd221f7e1a4b2fb481a019.
